server/models/schema.sql

```diff
@@ -317,51 +316,51 @@ CREATE TABLE `consumption_service` (
   FOREIGN KEY (`service_id`) REFERENCES `service` (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+DROP TABLE IF EXISTS `fee_center`;
 
-DROP TABLE IF EXISTS `cost_center`;
-
-CREATE TABLE `cost_center` (
-  `project_id` smallint(5) unsigned NOT NULL,
+CREATE TABLE `fee_center` (
   `id` smallint(6) NOT NULL AUTO_INCREMENT,
-  `text` varchar(100) NOT NULL,
-  `note` text,
+  `project_id` smallint(5) unsigned NOT NULL,
+  `label` varchar(100) NOT NULL,
+  `is_cost` tinyint(1) DEFAULT 0,
```

Comment:
Since you do a lot of queries on `is_cost`, as a performance improvement, you could put an INDEX on it.